pages/home/auth/secure-auth-production.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Secure Auth in Production"
+description: "How to secure and brand your auth flows in production"
+---
+
+# Secure and Brand the Auth Flow in Production
+
+To keep your users safe, Arcade.dev performs a session verification check when a tool is authorized for the first time. This check verifies that the user who is authorizing the tool is the same user who started the authorization flow, which helps prevent phishing attacks.
```

Comment:
> a session verification check

I like that phrasing a lot!

Reply:
@evantahler When I was building the UI, I ended up calling it "user verification" instead of "session verification". I think it's less jargony, what do you think?